lib/content-api-client.js

```diff
@@ -31,6 +31,9 @@ async function getRenderedRegionsByPageType({ accessToken, storeUrl, pageType })
             }),
         });
 
+        if (!response.data.data) {
+            return  { renderedRegions: []};
```

Comment:
This would probably fix the problem from a stencil-cli perspective, but do we know if there a more root cause as to why this `regions` object isnt populating? Is it just not being passed in properly?

Reply:
I had the same exact comment in the slack. I feel the issue is present somewhere else. `getRenderedRegionByPageType` has nothing to do with the theme. I have a feeling we are not passing down all the required params for the endpoint which could be the reason it's breaking.

Reply:
I created a follow up ticket to investigate this (posted over slack in threads). There is a 500 error from graphql. 
Meanwhile, can we unblock with this temporary solution? 
cc @jmwiese @jkanive @zvuki

Reply:
As long as we've captured the follow up work I think we can continue with this as a fix in the short term